app.js

```diff
@@ -8,12 +8,14 @@ require('dotenv').config()
 
 let indexRouter = require('./routes/index');
 let usersRouter = require('./routes/users');
+let tasksRouter = require('./routes/tasks');
 
 let app = express();
 
 // view engine setup
 app.set('views', path.join(__dirname, 'views'));
 app.set('view engine', 'jade');
+app.set('port', process.env.PORT || 5000);
```

Comment:
TBH I'm not sure why our app uses bin/www to execute, but the port is being set there by default already (it was set up by the express-generator)

Was this line necessary for something though?

Reply:
Ahh, I see; I didn't see where the port was being specified so I added to the app.js. I'll remove it